.ci/generate_bear_requirements.py

```diff
@@ -135,14 +144,14 @@ def get_gem_requirements(requirements):
 
 
 def get_npm_requirements(requirements):
-    npm_dependencies = {}
+    npm_dependencies = CommentedMap({})
 
     for requirement in requirements:
         package_name = requirement.package
         req_version = requirement.version
         if req_version:
             if req_version[0] not in ('<', '>', '~', '='):
-                req_version = "~" + req_version
+                req_version = '~' + req_version
```

Comment:
this isnt a change related to your problem, and ... it will be rewritten anyway in https://github.com/coala/coala-bears/pull/2561

Reply:
Okay (Y)